DYDX-Bots/Insolvent_Contract/README.md

```diff
@@ -0,0 +1,32 @@
+# Insolvent Contract Bot
+
+## Description
+
+This bot detects when the contract is insolvent. Insolvency refers to when the difference between `totalBorrowerDebtBalance` and `totalActiveBalanceCurrentEpoch` is greater than a set static threshold. The bot checks for both variables in every block.
+
+> Note: To set the static threshold to a desired amount, update the value of `THRESHOLD_AMOUNT` in `utils.ts`. Currently, it is set to `0`, and thus will detect **any** difference between the two varibles.
```

Comment:
```suggestion
> Note: To set the static threshold to a desired amount, update the value of `THRESHOLD_AMOUNT` in `utils.ts`. Currently, it is set to `0`, and thus will detect **all** insolvency cases.
```
I suggest this to help the client understand this direct link between this threshold and the insolvency detection.